src/api/entityResolution/types.ts

```diff
@@ -570,3 +570,7 @@ export enum MatchLevel {
   POSSIBLY_RELATED = 'POSSIBLY_RELATED'
   // These are known match level codes, some might be added later
 }
+
+export interface GetEntityResolutionServerStatusParams extends IDataSourceParams {}
+
+export type EntityResolutionServerStatus = 'up' | 'down';
```

Comment:
why not a boolean?

Reply:
In case we want to have more status, having an enum will make it easier (there is a low chance that it will happen, but just in case).